geppetto.js/geppetto-client/src/common/layout/LayoutManager.tsx

```diff
@@ -390,7 +390,9 @@ class LayoutManager {
       }
       case Actions.MAXIMIZE_TOGGLE:
         // FIXME: See https://github.com/MetaCell/geppetto-meta/issues/108
-        this.store.dispatch(updateWidget({ ...this.getWidget(action.data.node)}))
+        // fixed within netpyne-ui, the state is mismanaged, the widgets are not maximised
```

Comment:
@ddelpiano I would only mention that there is no widget update because maximization is a tabset property and remove the mention to #108

Reply:
also it would be preferable not to mention specific projects imo